dev/tools/controllerbuilder/scaffold/controller.go

```diff
@@ -34,29 +34,78 @@ const (
 	directControllerRelPath = "pkg/controller/direct"
 )
 
-func Scaffold(path string, cArgs *ccTemplate.ControllerArgs) error {
+func Scaffold(service, kind string, cArgs *ccTemplate.ControllerArgs) error {
```

Comment:
Nit: This is pre-existing, but we normally import alias with all lower case i.e. `cctemplate`